Dockerfile

```
@@ -0,0 +1,12 @@
+FROM node:16
+
+WORKDIR /app
+COPY . .
+
+RUN apt-get update && apt-get install jq gawk -y
+RUN cd nodejs 
+
+WORKDIR /app/nodejs
+RUN npm install && npm ci
```

```suggestion
RUN npm ci
```
`npm ci` installs the packages too but in a different way from `npm install`. No need to run both. Please read the docs.

---

yes and the error says exactly what's wrong:
```
`npm ci` can only install packages when your package.json and package-lock.json or npm-shrinkwrap.json are in sync. Please update your lock file with `npm install` before continuing.
```

---

ok , lets try